### ISSUE TITLE
Why the bootstrap 4 dependency?

### ISSUE THREAD

**Issue by igor-pavlichenko** [rsaenen/ngx-custom-validators#31]
It's specified in the dependencies as `"bootstrap": "^4.1.3",`, what is it used for here?

For some reason it's breaking my angular 9 project that uses bootstrap-sass 3 in legacy code. It's related to sass exports/imports.

**Comment by pedrotoch** [comment]
Same happens to me as soon as I update ngx-custom-validators to latest version. Also using bootstrap-sass 3.

**Comment by YZahringer** [comment]
Same issue, with warning on `npm audit`
````
WARN bootstrap@4.4.1 requires a peer of jquery@1.9.1 - 3 but none is installed. You must install peer dependencies yourself.

WARN bootstrap@4.4.1 requires a peer of popper.js@^1.16.0 but none is installed. You must install peer dependencies yourself.
````

**Comment by igor-pavlichenko** [comment]
@rsaenen you are using bootstrap for development/testing purposes in `/src/dev` so my guess is that it should be moved back to devDependencies

**Comment by rsaenen** [comment]
@igor-pavlichenko thanks a lot !

Released with `9.0.1`
